### PR TITLE
chore: add basic meta tags

### PR DESCRIPTION
**Pull request by kaf-lamed-beyt**
partial fix to #1

### CODE REVIEW

**Comment by kaf-lamed-beyt** [comment]
Hi @AdeboyeDN, kindly review and let me know what you think. 

Although, this is just a partial fix. I still want to proceed by adding the Twitter and Google metatags too. Let me know what you think when you get this.

File: theme.config.tsx [hunk 1]
```diff
@@ -1,53 +1,68 @@
-import { DocsThemeConfig, useConfig } from 'nextra-theme-docs'
-import { useRouter } from 'next/router'
-
-
+import React from "react";
```
Comment (AdeboyeDN) [inline theme.config.tsx]: So why did you import react?

File: theme.config.tsx [hunk 2]
```diff
@@ -1,53 +1,68 @@
...
+      };
     }
   },
 
+  head: () => {
+    const { asPath } = useRouter();
+    const { frontMatter } = useConfig();
+
+    return (
+      <React.Fragment>
```
Comment (AdeboyeDN) [inline theme.config.tsx]: I believe you don't need the `<React.Fragment>` here, this is an example from the previous link I sent you 
image
Read more here https://nextra.site/docs/docs-theme/theme-configuration#dynamic-tags-based-on-page

Comment (kaf-lamed-beyt) [reply theme.config.tsx]: I think `React.Fragment` and `<></>` are the same. But, I can change it, if it isn't necessary. 

...and I imported React so I can use the `Fragment` 

Kindly let me know what your thoughts are on this.

Comment (AdeboyeDN) [reply theme.config.tsx]: Using react isn't necessary you can change it we are focused on using nextjs fully.

Comment (kaf-lamed-beyt) [reply theme.config.tsx]: Great. I'll do that now

**Comment by kaf-lamed-beyt** [comment]
All done, @AdeboyeDN.

I added the twitter card and google meta attributes too. About the preview images, please when it is ready, kindly let me know so I can make the change.

**Pull request merged**